tests/integration/setup.go

```diff
@@ -0,0 +1,256 @@
+// Copyright 2023 Democratized Data Foundation
```

Comment:
praise: I like that this is being broken out of the main (and large) utils2.go file

suggestion: I think it might be worth splitting it up again, I do dislike 'setup' files in general, as they often reduce the cohesion of the codebase, moving stuff away from where it is used.  However here we only really have two things, and they are independent.  We could split this file up into `clients.go` and `db.go` (or similar names), this would keep the database stuff together, and the client stuff together, both out of the main utils.go, and avoid us creating a general purpose, lets shove everything in it, setup file that risks growing horribly in the future if/when we add more stuff that requires setup.

There is also an argument to be had that 'setup' here is misleading (particularly considering the large init func at the top of the file), as it doesn't actually setup up the database or clients, it hosts the functions that the utils2.go calls to set them up.

Reply:
I like that idea. I split them into `db.go` and `client.go` and put the init back into `utils2.go`

Reply:
I think you forgot to add+push the new files :) 

> and put the init back into utils2.go

Why did you chose to return the init?  I did picture that being split up too.

Reply:
Okay I see what you mean. I split the `init` logic but left the mutation variables in utils since they are not db or client specific.